client.go

```diff
@@ -46,7 +46,13 @@ func (c *Client) Exec(r Request) (*Response, error) {
 		}
 
 		if r.keepAlive() {
-			c.conn.(*net.TCPConn).SetKeepAlive(true)
+			switch c.network {
+			case "tcp":
+				c.conn.(*net.TCPConn).SetKeepAlive(true)
+				break
```

Comment:
`break` is not needed in Go at the end of case blocks. Could you please remove them?